2_process/src/calculate_toha.R

```diff
@@ -0,0 +1,153 @@
+# Functions to calculate daily TOHA for each lake
+# Edited `mda.lakes` functions.
+
+calculate_toha_per_lake <- function(target_name, nhdhr, nhdhr_data_fn, morphometry) {
```

Comment:
I am not reviewing the TOHA calcs with this PR (perhaps that review should be separate?)

But I did open up one of the three lakes that ran and see patterns like this:
![image](https://user-images.githubusercontent.com/2349007/79380717-6d739e80-7f26-11ea-962d-ad6184beb56e.png)

for `toha_nhdhr_120019107.csv`
I am assuming that means the total benthic area for the lake is 400203 and that the lake is very commonly fully within the water temperature thresholds. I did a spot check and this seems to be correct